packages/react/src/context/use-context-mutator.ts

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useContext, useRef } from 'react';
+import type { EvaluationContext } from '@openfeature/web-sdk';
+import { OpenFeature } from '@openfeature/web-sdk';
+import { Context } from '../common';
+
+export type ContextMutationOptions = {
+  /**
+   * Mutate the default context instead of the domain scoped context applied at the `<OpenFeatureProvider/>`.
+   * Note, if the `<OpenFeatureProvider/>` has no domain specified, the default is used.
+   * See the {@link https://openfeature.dev/docs/reference/technologies/client/web/#manage-evaluation-context-for-domains|documentation} for more information.
+   * @default false
+   */
+  defaultContext?: boolean;
```

Comment:
@wichopy @lukas-reining I've changed this from `setGlobal` to `defaultContext` for 2 reasons:

1 - I eventually want to return 2 functions from this hook, 1 for `set` and one for `merge`, so "set" seemed wrong
2 - we call this the `default` context mostly in reference to other domain-scoped context